appinventor/blocklyeditor/src/versioning.js

```diff
@@ -743,6 +743,28 @@ Blockly.Versioning.v17_translateComponentSetGetProperty = function(blockElem) {
     mutationElement.removeAttribute('yailtype');
 };
 
+/******************************************************************************
+ Upgrade screen names to use dropdown block.
+ ******************************************************************************/
+
+Blockly.Versioning.makeScreenNamesBeDropdowns = function () {
```

Comment:
Given that there isn't any configuration required for the operation, it seems like we don't need to wrap the upgrade function in a closure.